Providers/Scripts/2.4x-2.5x/Scripts/Tests/test_nxProviders.py

```diff
@@ -30,8 +30,8 @@ def opened_w_error(filename, mode="r"):
 try:
```

Comment:
Could this import block be moved before `opened_w_error` with the other imports?